src/sentry/models/platformexternalissue.py

```diff
@@ -12,7 +12,7 @@ class PlatformExternalIssue(Model):
     __core__ = False
 
     group = FlexibleForeignKey("sentry.Group", null=True, db_constraint=False, db_index=False)
-    project = FlexibleForeignKey("sentry.Project", null=True, db_constraint=False)
```

Comment:
Is it safe to make project non-nullable?

Since group_id wasn't an FK, there isn't a guarantee that every PlatformExternalIssue actually has a corresponding Group. Would someone be able to confirm if in practice every existing PlatformExternalIssue has a Group that still exists?

If not, I think this column will have to remain forever nullable.